SIAB/io/read_input.py

```
@@ -536,10 +567,51 @@ def skip_ppread(user_settings: dict):
+    Parameters
+    ----------
+    user_settings: dict
+        the user settings
+    
+    Returns
+    -------
+    None
+    """
+    # check if the shape assigned to orbitals is valid
+    shape2index = {rs["shape"]: i for i, rs in enumerate(user_settings["reference_systems"])}
+    for iorb, orb in enumerate(user_settings["orbitals"]):
```

Ruff

Loop control variable `iorb` not used within loop body

(B007)